### PR TITLE
Vmal

### PR DESCRIPTION
**Pull request by Malisevych**

### CODE REVIEW

File: RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java [hunk 1]
```diff
@@ -20,121 +20,151 @@
...
-        if (CollectionUtils.isNotEmpty(stores)) {
-            return stores;
+    public List<Store> getStoresByProductId(int productId, int userId) {
+
+        List<Store> storesInProduct = productDAO.getStoresByProductIdAndUserId(productId, userId);
+        List<Store> stores = new ArrayList<>();
+
+        if (!storesInProduct.isEmpty()){
+            for(Store getStore : storesInProduct) {
+                if(getStore.isEnabled()) {stores.add(getStore);}
```
Comment (Dyndyn) [inline RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java:118]: MAJOR At most one statement is allowed per line, but 2 statements were found on this line. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)

File: RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java [hunk 2]
```diff
@@ -20,121 +20,151 @@
...
-        List<Product> products = new ArrayList<Product>();
-        if (!getProducts.isEmpty()) {
-            for (Product getProduct : getProducts) {
-                if (getProduct.isEnabled()) {
-                    products.add(getProduct);
-                }
+        List<Product> products = new ArrayList<>();
+        if (!getProducts.isEmpty()){
+            for(Product getProduct : getProducts) {
+                if(getProduct.isEnabled()) {products.add(getProduct);}
```
Comment (Dyndyn) [inline RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java:44]: MAJOR At most one statement is allowed per line, but 2 statements were found on this line. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)

**Comment by Dyndyn** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [ProductController.java#L25](https://github.com/IF-072/Product-Tracker/blob/95f36e0119b6e755906e93189e403a8a1de530d9/RESTserver/src/main/java/com/softserve/if072/restservice/controller/ProductController.java#L25): Remove this unused import 'java.util.ArrayList'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [ProductService.java#L7](https://github.com/IF-072/Product-Tracker/blob/95f36e0119b6e755906e93189e403a8a1de530d9/RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java#L7): Remove this unused import 'org.apache.commons.collections.CollectionUtils'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java (hunk 2 above)
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java:44]: Please format the code

File: RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java (hunk 1 above)
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java:118]: please format the code

**Pull request merged**